### ISSUE TITLE
Update from within?

### ISSUE THREAD

**Issue by gingerbeardman** [Fledge68/WiiFlow_Lite#147]
I see some old information about this here: https://github.com/Fledge68/WiiFlow_Lite/blob/master/docs/versions.txt

Was it removed?

Would you consider reintroducing it?

**Comment by Fledge68** [comment]
yes wiiflow could update from within. but no point in re adding it. there wont be many (if any) updates from here on out.

**Comment by eku** [comment]
> but no point in re adding it. there wont be many (if any) updates from here on out.

Are you serious?

**Comment by gingerbeardman** [comment]
If that's the case, do we need to talk about assigning contributors with enough permissions to keep going with the project?
